support/dynamics/gp/questions-home-pages-features.md

```diff
@@ -228,125 +196,102 @@ This article contains answers to frequently asked questions about the home pages
 
-- **Q28: How do I remove the Outlook, Metrics, or To Do: Reminders areas of the home page for new users?**
-
-  A28: To remove the **Outlook**, **Metrics**, or **To Do: Reminders** areas of the home page for new users, follow these steps:
-
-  1. Access the System Preferences window. To do this, click **Microsoft Dynamics GP**, click **Tools**, click **Setup**, click **System**, and then click **System Preferences**.
-
-  2. In the **Home Page Defaults** section, click to clear the **Load To Do: Reminders**, **Load Metrics**, or **Load Microsoft Office Outlook** check boxes.
-
-    > [!NOTE]
-    > This does not affect the existing users. This affects only new users who are created after you change these settings. See question and answer 13 if you have to disable the Outlook part of the home page for the existing users.
+- **Q23: How do I remove the Outlook, Metrics, or To Do: Reminders areas of the home page for new users?**
 
-- **Q29: How do I add a Smartlist Builder object to the My Reports section on the Microsoft Dynamics GP 10.0 homepage?**
+  A23: To remove the **Outlook**, **Metrics**, or **To Do: Reminders** areas of the home page for new users, follow these steps:
 
-  A29: To add a Smartlist Builder object to the My Reports section of the homepage, follow the steps:
+  1. Access the System Preferences window. To do this, select **Microsoft Dynamics GP** > **Tools** > **Setup** > **System** > **System Preferences**.
 
-    1. Start Microsoft Dynamics GP 10.0 as the user who wants the SmartList Builder on their homepage.
-    2. Click **Microsoft Dynamics GP,** point to **Tools**, point to **SmartList Builder**, and then **click SmartList Builder**.
-    3. In the **Smartlist ID** box, type the ID of the Smartlist Builder that you want to add to the My Reports section of the homepage.
-    4. In the **Series** list, note the option that is listed, and then exit SmartList Builder.
-    5. In the Navigation Pane, click the series that you noted from step 4, and then click **Report List**.
-    6. After the reports display, locate the name of the SmartList Builder object, and then click to check the report name.
-    7. In the My Reports section, click **Add to**, type a report name, and then click **OK**  
-    8. In the Navigation Pane, click **Home**, expand My Reports, and then verify the report is now listed under the My Reports section.
+  2. In the **Home Page Defaults** section, clear the **Load To Do: Reminders**, **Load Metrics**, or **Load Microsoft Office Outlook** checkboxes.
 
     > [!NOTE]
-    > If the report is not added, refresh the homepage or exit and restart Microsoft Dynamics GP as the same user.
+    > This doesn't affect the existing users. This affects only new users who are created after you change these settings. See question and answer 10 if you have to disable the Outlook part of the home page for the existing users.
```

Comment:
Hi @theley502, for line 292 I replaced "question and answer **13**" with "question and answer **10**".